### ISSUE TITLE
bun v1.1.10 [maybe Windows?] cannot install packages with @swc/types

### ISSUE THREAD

**Issue by cpakken** [oven-sh/bun#11413]
### What version of Bun is running?

1.1.10+5102a9443

### What platform is your computer?

Microsoft Windows NT 10.0.19045.0 x64

### What steps can reproduce the bug?

Clean bun install `bun init`

`bun a @vitejs/plugin-react-swc`
or
`bun a @esbuild-plugin-swc-minify`
or
Anything dependent on `swc/types` https://www.npmjs.com/package/@swc/types?activeTab=dependents

I tried `bun i` after upgrading bun from 1.1.8 to 1.1.10 and I can't install `@vitejs/plugin-react-swc` like before. I suspect its the changes made to workspace installs.

![image](https://github.com/oven-sh/bun/assets/61718569/72d3b3a5-0257-4309-a2fd-059a71e47174)

**Comment by cckn** [comment]
This issue occurs with @swc/core version 1.5.11. To resolve this issue:

1. Install @swc/core version 1.5.7:
```bash
burn install @swc/core@1.5.7
```

2. If @swc/core is already installed, fix it to version 1.5.7. Remove any caret (^) if present.
```json
"dependencies": {
  "@swc/core": "1.5.7"
}
```
For more details, refer to the https://github.com/swc-project/swc/issues/8988

**Comment by cpakken** [comment]
I see, so not a bun issue
